snapcraft/internal/lxd.py

```diff
@@ -120,6 +146,39 @@ def _wait_for_network(self):
         logger.info('Network connection established')
 
 
+class Cleanbuilder(Containerbuild):
+
+    def __init__(self, *, output, source, project_options,
+                 metadata=None, remote=None):
+        container_name = petname.Generate(3, '-')
```

Comment:
This gets `snapcraft-` prepended to it, which makes for quite a long container name. Would it be worth shortening this to 2?